src/app/spotlight-listing/spotlight-listing.page.html

```diff
@@ -0,0 +1,21 @@
+<app-application-header [config]="headerConfig"></app-application-header>
+
+<ion-content [fullscreen]="true">
```

Comment:
(Optional, for future considerations, Please note) the html looks similar to improvements page, we can try to shift those common items to a common component

Reply:
+ for logic

Reply:
okay will do it later